guides/common/modules/proc_installing-flatpak-applications-on-project-hosts.adoc

```diff
@@ -0,0 +1,35 @@
+[id="installing-flatpak-applications-on-project-hosts"]
+= Installing Flatpak applications on {Project} hosts
+
+Use the command line to install selected applications from the enabled Flatpak remotes.
+
+.Prerequisites
+* Install Flatpak.
+ifdef::satellite[]
+* Enable the {Team} Flatpak remote.
+endif::[]
+
+.Procedure
+. On the managed host, add your {ProjectServer} as a Flatpak remote:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ flatpak remote-add --authenticator-name=org.flatpak.Authenticator.Oci katello oci+https://_{foreman-example-com}_/
+----
+. Log in using Podman:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ podman login
+----
+. Install your application.
+For example, to install the Mozilla Firefox Flatpak:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ flatpak install firefox
+----
+
+.Additional resources
+* For more information about launching Flatpak applications, see {RHELDocsBaseURL}9/html/administering_the_system_using_the_gnome_desktop_environment/assembly_installing-applications-using-flatpak_administering-the-system-using-the-gnome-desktop-environment#launching-flatpak-applications_assembly_installing-applications-using-flatpakassembly_installing-applications-using-flatpak_using-the-desktop-environment-in-rhel-8#launching-flatpak-applications_assembly_installing-applications-using-flatpak[Launching Flatpak applications].
```

Comment:
This link looks broken. Can you look into it @bangelic?

Reply:
Fixed and pushed.